vw_golf_mk4.dbc

```diff
@@ -1121,7 +1121,7 @@ BO_ 208 Lenkhilfe_3: 6 XXX
  SG_ LH3_BLWValid : 45|1@1+ (1,0) [0|1] "" XXX
  SG_ LH3_Lenkungstyp : 46|2@1+ (1,0) [0|3] "" XXX
 
-BO_ 978 Lenkhilfe_2: 8 XXX
+BO_ 978 Lenkhilfe_2: 7 XXX
```

Comment:
@jyoung8607 FYI in case you want to double check this. the PQ test route we have in openpilot was hitting this

Reply:
Comparing to documentation, your change appears correct.